service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/MigrateCaseController.java

```diff
@@ -35,19 +35,8 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
         Map<String, Object> data = caseDetails.getData();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        if (defaultIfNull(caseData.getFamilyManCaseNumber(), "").equals("SA20C50019")) {
-            List<Element<Child>> children = caseData.getChildren1();
-
-            for (Element<Child> child : children) {
-                child.getValue().setFinalOrderIssued(null);
-                child.getValue().setFinalOrderIssuedType(null);
-            }
-
-            caseData.getOrderCollection().remove(1);
-            data.put("orderCollection", caseData.getOrderCollection());
-            data.put("children1", children);
+        if (defaultIfNull(caseData.getFamilyManCaseNumber(), "").equals("SA20C50004")) {
```

Comment:
if you flip the equals and then you won't need the `defaultIfNull`

Reply:
this is true